superset/assets/javascripts/explorev2/components/ExploreViewContainer.jsx

```diff
@@ -74,6 +75,25 @@ class ExploreViewContainer extends React.Component {
   }
 
   render() {
+    const errors = [];
```

Comment:
what do you think of pulling this out of the render method in it to it's own method? i prefer keeping the render methods free of as much logic as possible.